Algorithms/Searching/Lower_Bound.cpp

```diff
@@ -0,0 +1,55 @@
+//Name : Shivam Singhal
+//Email ID : shivamsinghal0610@gmail.com
+
+/*program to find minimum position of element lesser than given value 
+in sorted fashion*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
```

Comment:
Please go through [guidelines](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) before making any pull request.@shivhek25

Reply:
Yes, I did already. I didn't get what you pointed here.

Reply:
I hope you have seen instructions for header file.

Reply:
Yes, that was added recently, I think. XD
Made the change.